_posts/2021-02-22-hhvm-4.98.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "HHVM 4.98"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.98 is released! This release marks the end of support for 4.92;
+HHVM 4.93&ndash;4.97 remain supported, as do the 4.56 and 4.80 LTS releases.
+
+# Highlights
+
+- Minor improvements to various typechecker error messages.
+
+# Breaking Changes
+
+- The typechecker now correctly reports errors when a class inherits multiple
+  conflicting definitions of the same non-abstract constant from its parent
+  class/interfaces/traits. These had already been runtime errors, but previously
+  wouldn't be caught by the typechecker.
+- Classes that implement
+  [`dynamic`](https://docs.hhvm.com/hack/built-in-types/dynamic)
+  now have stricter requirements for their properties
+  ([examples](https://github.com/facebook/hhvm/commit/4b14f6e8a817acc4a03a6917525afca10957f9de#diff-d2883cc8990002e89fc8a04f1db18691a507699dd1b849ef4f8d535b79322bd1)).
```

Comment:
FWIW implementing `dynamic` is still an experimental feature (the tests in the commit opt-in with `--enable-sound-dynamic-type`). This isn't a breaking change.

Reply:
Thanks, I'll remove it.